### ISSUE TITLE
Pads do not work on iOS 10 Safari

### ISSUE THREAD

**Issue by cbertelli** [ether/etherpad-lite#4703]
**Describe the bug**

**To Reproduce**
Open a document on Safari (this is an old version, I understand). The page is completely blank
**Expected behavior**
Visibile text and interface

**Tablet**
 - OS: iOS
 - Browser safari
 - Version iOS 10

**Comment by JohnMcLear** [comment]
Does etherpad even support this version of iOS?

Did you try latest develop?

**Comment by packardone** [comment]
Do you really mean iOS 10?  The current version is 14.  10 is from 2016.  Etherpad works well cross browsers on at least 12, 13, and 14.

**Comment by cbertelli** [comment]
I understand it's old. Paper still works on it. Do you have any suggestions about plugins or settings that could be particularly difficult for such an old safari to handle. I'm using version 1.8.7. Is there some parameter that temporary disables them at start?

**Comment by JohnMcLear** [comment]
I think we just decided to drop such old browsers because we want to look to the future and it was a trade off we made dropping IE and old Safari's.  I'm not sure what Apple are doing supporting installing modern Safari on older devices but I'd look in that area for a fix.

**Comment by JohnMcLear** [comment]
I'm gonna close as wontfix

**Comment by cbertelli** [comment]
I think it's reasonable, but please keep it open for a short time, I'm
going to check what's the incompatible script because HackMD,
CodiMD/HedgeDoc, Paper, StackEdit all work. It's strange and I think there
are recent changes or plugins (the suggested ones) that hamper it. I have
no experience in debugging javascript on iPad but I think there are not so
many devices around that cannot be upgraded to at least iOS 12.

**Comment by JohnMcLear** [comment]
@cbertelli yeah we modernized a lot of the codebase recently.

**Comment by webzwo0i** [comment]
just wanted to add my opinion:
I don't mind supporting older browsers (or hardware) when it does not stop us from using new JS or browser features. IE if there are `if (browser.safari && navigator.platform...` branches it's okay, as they are very self-explanatory. It doesn't matter if one or another UI feature is unavailable, as long as we can keep track of it. So my biggest concern here is, that if we don't have browser test coverage, they will start failing again very soon, because we only use a limited amount of different browsers during development.

I'm not gonna work on this in the forseeable future, but in theory we could add a cronjob that is run weekly against develop that uses saucelabs/browserstack/cypress with a whole lot of browser combinations. (Or split it, like "old firefox", "old safari", "old android" over the week). However, I don't see much requests like this, even not for IE which we dropped some time ago.

**Comment by rhansen** [comment]
Once #4820 is done it should be straightforward to add a babel transpile step so that we can use modern JS but still support older browsers.
